javadoc/src/test/java/org/apache/cxf/xjc/javadoc/JavadocPluginTest.java

```diff
@@ -124,11 +124,11 @@ public void testDocumentationOnPropertiesIsOverwrittenByJAXBBindings() throws Ex
 
         CompilationUnit compilationUnit = parseSourceFile("ComplexTypeWithDocumentedProperties.java");
         Javadoc fieldJavadoc = getJavadocOfField(compilationUnit, "documentedElement");
-        assertThat(fieldJavadoc, not(javadocContains("Some documentation of element")));
+        assertThat(fieldJavadoc, javadocContains("Some documentation of element"));
```

Comment:
Seems like https://github.com/eclipse-ee4j/jaxb-ri/issues/1708 was fixed and test needs adjustment now